packages/action-menu/src/__tests__/ActionMenuModule.test.ts

```diff
@@ -2,13 +2,10 @@ import type { DependencyManager, FeatureRegistry } from '@aries-framework/core'
 
 import { Protocol } from '@aries-framework/core'
 
-import {
-  ActionMenuApi,
-  ActionMenuModule,
-  ActionMenuRepository,
-  ActionMenuRole,
-  ActionMenuService,
-} from '@aries-framework/action-menu'
+import { ActionMenuRepository } from '../repository'
+import { ActionMenuService } from '../services'
+
+import { ActionMenuApi, ActionMenuModule, ActionMenuRole } from '@aries-framework/action-menu'
```

Comment:
Should we not import all directly from the package in a unit test inside the package? I mean, replace `@aries-framework/action-menu` with relative paths?

Reply:
Yes could do it like that!